### PR TITLE
LPD-23693 set allowScriptContentToBeExecutedOrIncluded default value as false

### PR DESCRIPTION
**Pull request by MarinhoFeliphe**

### CODE REVIEW

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

**Comment by MarinhoFeliphe** [comment]
ci:test:object

**Comment by MarinhoFeliphe** [comment]
ci:test:workflow

**Comment by MarinhoFeliphe** [comment]
ci:test:workflow-metrics

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

**Comment by MarinhoFeliphe** [comment]
ci:test:workflow

**Comment by MarinhoFeliphe** [comment]
ci:test:object

**Comment by MarinhoFeliphe** [comment]
ci:test:workflow-metrics

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java [hunk 1]
```diff
@@ -1660,35 +1666,43 @@ public void testAddObjectActionWithSystemObject() throws Exception {
...
-			LocalizedMapUtil.getLocalizedMap(RandomTestUtil.randomString()),
-			LocalizedMapUtil.getLocalizedMap(RandomTestUtil.randomString()),
-			RandomTestUtil.randomString(),
-			ObjectActionExecutorConstants.KEY_GROOVY,
-			ObjectActionTriggerConstants.KEY_ON_AFTER_UPDATE,
-			UnicodePropertiesBuilder.put(
-				"script", "println \"Hello World 2\""
-			).build(),
-			false);
+		try (Closeable closeable =
```
Comment (linolaoj) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java:1689]: Hi @MarinhoFeliphe , do you think we can use only one `try` here?

**Review by linolaoj — CHANGES_REQUESTED** [review]

**Comment by MarinhoFeliphe** [comment]
Resent here: https://github.com/liferay-objects/liferay-portal/pull/3313

**Pull request closed without merging**